misc/nuitka-package-config-schema.json

```diff
@@ -9,42 +9,38 @@
                 "pattern": "^[A-Za-z0-9\\.\\_\\-]+$"
             },
             "data-files": {
-                "items": {
-                    "type": "object",
-                    "additionalProperties": false,
-                    "properties": {
-                        "dest_path": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                    "dest_path": {
+                        "type": "string"
+                    },
+                    "dirs": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "patterns": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    },
+                    "control_tags": {
+                        "type": "string",
+                        "pattern": "(([a-zA-Z]|[0-9]|_)|and|is|not| )"
```

Comment:
we should also have `or` here.

Reply:
also, shouldn't the "node" be called "control_tag" since there can only be one now?

Reply:
I have deleted this regex. It becomes to complex.